api/app/routers/hfi_calc.py

```diff
@@ -361,6 +368,42 @@ async def get_fire_centres(response: Response):
         raise
 
 
+@router.get('/add-station/{fire_centre_id}', response_model=HFIAddStationOptionsResponse)
```

Comment:
I think this endpoint is redundant - since we already have well tested endpoints that provide all this information.